lib/revamp/cli.rb

```diff
@@ -80,6 +80,7 @@ def parser
       p.option :filenames, 'Files which will be processed', default: []
       p.option :verbose, 'Should print all information, by default: false', default: false
       p.option :cleanup, 'Should temporary files be cleaned up, by default: true', default: true
+      p.option :clobber, 'Should overwrite output converted packages, by default: true', default: true
```

Comment:
Wouldn't it be better to set clobber default to false, so that revamp by default would not overwrite files?